xml/Makefile.am

```diff
@@ -141,6 +141,21 @@ api/api-result-%.rng: $(API_files) Makefile.am
 	echo '      <optional>' >> $@
 	echo '        <choice>' >> $@
 	for rng in $(API_request_base); do $(top_srcdir)/xml/best-match.sh api/$$rng $(*) $(@) "          " || :; done
+	echo '          <element name="version">' >> $@
+	echo '            <choice>' >> $@
+	echo '              <group>' >> $@
+	echo '                <attribute name="program"> <text /> </attribute>' >> $@
+	echo '                <attribute name="version"> <text /> </attribute>' >> $@
+	echo '                <attribute name="build"> <text /> </attribute>' >> $@
+	echo '                <attribute name="features"> <text /> </attribute>' >> $@
+	echo '              </group>' >> $@
+	echo '              <group>' >> $@
+	echo '                <attribute name="program"> <text /> </attribute>' >> $@
+	echo '                <attribute name="version"> <text /> </attribute>' >> $@
+	echo '                <attribute name="author"> <text /> </attribute>' >> $@
+	echo '              </group>' >> $@
+	echo '            </choice>' >> $@
+	echo '          </element>' >> $@
```

Comment:
I think the echo block has gotten complicated enough that we should probably move status and version to their own files. (That would give us a chance to bump to 2.0 as well.)